cypress.config.js

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const { defineConfig } = require('cypress');
+
+module.exports = defineConfig({
```

Comment:
IMO, I think if we are adding this already we should create a `cypress` folder in [test folder](https://github.com/opensearch-project/OpenSearch-Dashboards/tree/main/test), move this [spec](https://github.com/opensearch-project/OpenSearch-Dashboards/pull/5725/files#diff-b2eea1186135b946e8d070f62b93e527ddc1f56ae90e8ae295630efd6c80ef55) to  that folder.

Sort of how like selenium does it where it defines the config at the functional folder level and adds subdirectories below that.